scss/os/_overrides_tuieditor.scss

```diff
@@ -367,6 +367,8 @@
   //////////// EDITOR POPUP WINDOWS //////////
   .tui-popup-wrapper,
   .tui-editor-popup {
+    z-index: 500;
```

Comment:
We prefer to base `z-index` values off of `$u-zindex-base`, to help enforce consistent ordering across our SCSS. That's set to 500, so you should be good dropping it in here.

Reply:
I changed it to use that scss variable.